modules/contact/include/constraints/ComputeWeightedGapLMMechanicalContact.h

```diff
@@ -72,10 +80,16 @@ class ComputeWeightedGapLMMechanicalContact : public ADMortarConstraint
 
   /// The value of the gap at the current quadrature point
   ADReal _qp_gap;
+  /// The value of the LM at the current quadrature point
+  Real _qp_factor;
```

Comment:
Are you adding mechanical contact tests for 3D?

Reply:
I haven't added any, I can look at adding a couple simple ones but that's definitely something I could have done better.